contracts/contracts/URaidenEIP712HelperContract.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity ^0.4.17;
+
+contract URaidenEIP712HelperContract {
```

Comment:
###Naming Suggestion###

We call the microraiden contract `RaidenMicroTransferChannels.sol` so the library contract should also use `micro` and not `u`. So I would suggest something like `MicoRaidenEIP712Helper`. or `MicroRaidenEIP712Signer`. No need to keep the `contract` part in the name. We know it's a contract.

Reply:
`MicoRaidenEIP712Helper` it is then. I tried a couple of names, but I knew you are better at this and will probably find a good one hahah.